File: desktop/src/com/mljoke/rajon/desktop/DesktopLauncher.java
package com.mljoke.rajon.desktop;

import com.badlogic.gdx.Files;
import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.mljoke.rajon.Core;
import com.procedural.world.PBRTestAPP;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		config.title = "Rajon";
		config.fullscreen = false;
		config.width = 1440;
		config.height = 900;
		config.useHDPI = true;
		config.addIcon("icon.png", Files.FileType.Internal);
		//
		new LwjglApplication(new Core(), config);
//		new LwjglApplication(new PBRTestAPP(), config);
	}
}

File: desktop/src/com/mljoke/rajon/desktop/DesktopLauncher.kt
package com.mljoke.rajon.desktop

import com.badlogic.gdx.Files
import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.mljoke.rajon.Core

object DesktopLauncher {
    @JvmStatic
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration().apply {
            title = "Rajon"
            fullscreen = false
            width = 1600
            height = 900
            //useHDPI = true
            addIcon("icon.png", Files.FileType.Internal)
        }
        LwjglApplication(Core(), config)
    }
}